VideoLocker/src/androidTest/java/org/edx/mobile/test/http/SegmentTests.java

```diff
@@ -25,10 +28,13 @@ private void initTracker() {
         if (segment == null) {
             // create mocked instance of SegmentTracker
             tracker = Mockito.mock(ISegmentTracker.class);
```

Comment:
@aleffert ic. you just switch the implementation for testing.  i think that's the reason why to use Mock object/Mockito.